app/view_models/auctions_index_view_model.rb

```diff
@@ -1,29 +1,30 @@
-class AuctionsIndexViewModel < Struct.new(:current_user, :auctions_query)
-  def auctions
-    @auctions ||= auctions_query.map { |auction| AuctionViewModel.new(current_user, auction) }
+class AuctionsIndexViewModel
+  attr_reader :auctions, :current_user
+
+  def initialize(auctions:, current_user:)
+    @auctions = auctions.to_a
```

Comment:
this one took us a bit to figure out -- it's because the collection based on from the controller (`AuctionQuery.new.public_index`)  is not actually a collection, it's a query. And it doesn't fire until you map over it or call `to_a`